src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.String.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers.Binary;
+
+namespace System.Security.Cryptography.Encoding.Tests.Cbor
+{
+    internal ref partial struct CborReader
+    {
+        private static readonly System.Text.Encoding s_utf8Encoding = System.Text.Encoding.UTF8;
```

Comment:
This is the "replace invalid data" version. Do we want that? Or `new UTF8Encoding(false, true)`?

Reply:
It depends on the requirements I guess. I presume we have the luxury to be rigid here given it's not a human-readable format?